src/dashboard/public/styles.css

```
@@ -1227,3 +1227,32 @@ textarea { resize: vertical; min-height: 80px; }
     .dash-kpis { grid-template-columns: 1fr 1fr; }
     .dash-module-list { grid-template-columns: 1fr; }
 }
+
+/* ── Shop item image upload ───────────────────────────────────── */
+.shop-img-upload-row { display: flex; align-items: center; gap: .75rem; }
+.shop-img-preview-wrap { width: 64px; height: 64px; display: flex; align-items: center; justify-content: center; border: 1px solid var(--border); border-radius: 8px; background: var(--surface-2, #1e1e2e); flex-shrink: 0; }
+
+/* ── Activity item images grid ───────────────────────────────── */
+.activity-img-grid { display: flex; flex-direction: column; gap: 1.25rem; margin-top: .5rem; }
+.activity-img-category { }
```

Stylelint (17.12.0)

[error] Empty block (block-no-empty)

(block-no-empty)